rclrs/src/vendor/rcl_interfaces/msg.rs

```diff
@@ -404,6 +404,7 @@ pub mod rmw {
         pub description: rosidl_runtime_rs::String,
         pub additional_constraints: rosidl_runtime_rs::String,
         pub read_only: bool,
+        pub dynamic_typing: bool,
```

Comment:
Does this break backwards compatibility with humble?

Reply:
Humble does have a `dynamic_typing` field as well (https://github.com/ros2/rcl_interfaces/blob/humble/rcl_interfaces/msg/ParameterDescriptor.msg#L25), so this should be ok.